extensions/core/runtime/src/main/java/io/quarkiverse/cxf/transport/VertxHttpServletRequest.java

```diff
@@ -389,7 +385,12 @@ public Enumeration<String> getHeaders(String name) {
     public int getIntHeader(String name) {
         LOG.trace("getIntHeader");
         String v = getHeader(name);
-        return v == null ? -1 : Integer.parseInt(v);
+        try {
+            return v != null ? Integer.parseInt(v) : 0;
+        } catch (NumberFormatException nfe) {
+            LOG.error(nfe);
+        }
+        return 0;
```

Comment:
This change breaks the contract defined in javax.servlet.http.HttpServletRequest.getIntHeader(String). The old implementation was correct IMO. What is GH Code scanning complaining about?

Reply:
Thanks for reviewing @ppalaga .  I wasn't completely sure on how to handle this one either, happy to revert it back to the original state -- here's the finding:  https://github.com/quarkiverse/quarkus-cxf/security/code-scanning/60

And the full list of findings is here in case you come across anything else:
https://github.com/quarkiverse/quarkus-cxf/security/code-scanning?query=pr%3A649+tool%3ACodeQL+is%3Aopen+

Please feel free to give a thumbs up if we should revert to the original code!

Reply:
I took a closer look at `javax.servlet.http.HttpServletRequest.getIntHeader(String)` and you are indeed correct, the change was breaking the contract.  I just reverted that to the original state.  Thanks again for the review!